data/hai/hai reveal 4 middle.txt

```diff
@@ -843,17 +843,17 @@ mission "Hai Reveal [A06a] The Calm"
 			label accepted
-			`	"Excellent," says Remington, standing up and flicking over the pickup details to your pad. "Check in with Terry on your way; she may have updated scouting information on their precise placement. Now I have another meeting to get to momentarily though, but thank you Captain. It's been a pleasure seeing you again."`
-			`	He gives a meaningful look to Alondo on his way out, but exactly what its meaning is is not clear to you. Alondo's posture relaxes somewhat as he briefly gets up and quietly closes the door.`
-			`	"Now that that's over with, we have business. I have a lead on this symbol you wanted investigating. I still wish you'd tell me why you're looking for it, but for now, I'll continue to accept that it's important." he fishes around in a small satchel bag that usually carries his pad and unexpectedly pulls out a piece of actual physical paper. The text on it reads, 'Adjunct Research Library, Navy Intelligence Observatory, Moscow. Room number: 3017A19'.`
-			`	"Fortunately, I have been able to not only find you someone to speak to, but also find you a cover for being there in the first place. Xilin has decided he needs a trusted aide to stay on top of things here. Especially as Mountaintop progresses. So far he's found the staff provided him on Mountaintop to be... lacking. I get the sense from him that he's expecting things to get more complicated, not less; though he hasn't said as such in quite so many words just yet. So, you get to go to Earth to pick up his chosen assistant, and while you're there you can go and speak to a certain Ms. Giti Gilani. She's an expert in the history of space flight and related archaic topics. Meanwhile the aide you'll be picking up is a young man called Batu Cegeen. You'll have to land at the Ulaanbaatar spaceport for him and travel locally for your business with Ms. Gilani."`
+			`	"Excellent," says Remington, standing up and flicking over the pickup details to your pad. "Check in with Terry on your way; she may have updated scouting information on their precise placement. Now I have another meeting to get to momentarily, but thank you, Captain. It's been a pleasure seeing you again."`
+			`	He gives a meaningful look to Alondo on his way out, but exactly what its meaning is, is not clear to you. Alondo's posture relaxes somewhat as he briefly gets up and quietly closes the door.`
+			`	"Now that that's over with, we have business. I have a lead on this symbol you wanted investigating. I still wish you'd tell me why you're looking for it, but for now, I'll continue to accept that it's important." he fishes around in a small satchel bag that usually carries his pad and unexpectedly pulls out a piece of physical paper. The text on it reads, 'Adjunct Research Library, Navy Intelligence Observatory, Moscow. Room number: 3017A19'.`
```

Comment:
I suppose the paperless office came 1000 years later than people thought it would. It does sound better, and because the Endless Sky world isn't really all that different from ours, we need more of these little things that make it different.